wheel/src/lib.rs

```diff
@@ -44,6 +77,10 @@ pub async fn bootstrap_wheel(
         tokio::spawn(async move { worker.run().await });
     }
 
+    test_prometheus();
+    let listen_addr: String = format!("{}:{}", config.prometheus.host, config.prometheus.port);
```

Comment:
```suggestion
    let listen_addr = format!("{}:{}", config.prometheus.host, config.prometheus.port);
```

Please remove unnecessary type identifier.